haiku-driver.groovy

```diff
@@ -66,7 +67,9 @@ def parse(String description) {
                     } else {
                         events << createEvent(name: "switch", value: "on")
                     }
-                    events << createEvent(name: "level", value: values[4])
+                    level = Math.round(values[4].toInteger() * 6.25)
+                    log.debug "Using new level ${level}"
```

Comment:
Nitpick, but I don't think this log is needed? `sendLightLevelCommand` already logs the level being used.

Reply:
Fair point. Left over from my own debugging. Will remove it.